core/arch/riscv/include/tee/teeabi_opteed.h

```diff
@@ -0,0 +1,21 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright 2023 NXP
+ * Copyright (c) 2014, Linaro Limited
+ */
+
+#ifndef TEEABI_OPTEED_H
+#define TEEABI_OPTEED_H
+
+/*
+ * Issued when returning from "std_abi" or "fast_abi" vector
+ *
+ * Register usage:
+ * a0	ABI Function ID, TEEABI_OPTEED_RETURN_CALL_DONE
+ * a1-4	Return value 0-3 which will passed to normal world in a0-3
+ */
+#define TEEABI_OPTEED_FUNCID_RETURN_CALL_DONE		5
```

Comment:
It should not only copy `TEEABI_OPTEED_FUNCID_RETURN_CALL_DONE` here.
Other definitions should be copied also:

```
#define TEEABI_OPTEED_FUNCID_RETURN_ENTRY_DONE		0
#define TEEABI_OPTEED_FUNCID_RETURN_ON_DONE		1

......
```